autoload/tagbar.vim

```diff
@@ -1967,7 +1967,12 @@ function! s:PrintKinds(typeinfo, fileinfo) abort
             endif
 
             let padding = g:tagbar_show_visibility ? ' ' : ''
-            call add(output, foldmarker . padding . kind.long)
+            let tag_count = ' (' . len(curtags) . ')'
+            if g:tagbar_show_tag_count
```

Comment:
```suggestion
            if g:tagbar_show_tag_count
                let tag_count = ' (' . len(curtags) . ')'
```

As long as we have the feature behind a flag we might as well save the processing time generating the string if we're not going to use it.

Reply:
fixed